helpdesk-utility/src/main/java/org/symphonyoss/symphony/bots/utility/message/EntityBuilder.java

```diff
@@ -11,13 +11,13 @@
  * <p>
  * Created by rsanchez on 12/01/17.
  */
-public class EntityBuilder {
+public class  EntityBuilder {
 
   private static final ObjectMapper MAPPER = new ObjectMapper();
 
   private Map<String, Object> content = new LinkedHashMap<>();
 
-  private EntityBuilder() {
+  protected EntityBuilder() {
```

Comment:
You shouldn't update the class scope. Please use the 'createEntity' static method to instantiate this class.

Reply:
ok